bullet-featherstone/src/SimulationFeatures.hh

```diff
@@ -36,6 +37,8 @@ struct SimulationFeatureList : gz::physics::FeatureList<
 > { };
 
 class SimulationFeatures :
+    public CanWriteExpectedData<SimulationFeatures,
```

Comment:
is this happenning in `featherstone` too ? I can reproduce the issue in `bullet` but this one I believe it's fine, is there any other reason to add this here?

Reply:
No, this is not happening on `featherstone`, but inheriting from `CanWriteExpectedData` adds a compile-time check that the class implements `Write` for the expected data type. It's not strictly necessary.